apps/src/applab/DesignToggleRow.jsx

```diff
@@ -131,11 +137,12 @@ module.exports = React.createClass({
     }
 
     return (
-      <div className="justify-contents">
+      <div className={this.props.hideToggle ? 'rightalign-contents' : 'justify-contents'}>
```

Comment:
maybe worth checking if justify-contents still does the right thing? I would expect display: none; to mess up the text alignment, but I would expect visibility: hidden; not to change it.